egs/wsj/s5/steps/data/gen_wav_sample.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+from __future__ import print_function
```

Comment:
I wanted him to delete gen_wav_sample.py from the PR anyway-- it's just something he used for testing, I think.

Reply:
Oh，I think some people like me may want to listen some samples after doing adding reverbration or adding noises. Or do you think it should be moved to utils directory?

Reply:
The purpose of the script wasn't clear, and the script doesn't clearly explain what its arguments mean (i.e. what the interface is).
People may want to do that, but I don't want you to mix that up with the bug-fix commit.  You are making this too complicated. Talk to David by email  (David Snyder <david.ryan.snyder@gmail.com>) about this possibility, but in any case, do it as a separate commit.

Reply:
OK, I will remove it from this PR.